python/data/chap3/README.rst

```diff
@@ -0,0 +1,28 @@
+Chapter 3 - Logical Record Syntax
+=================================
+
+Testfiles for the general layout of (Explicitly) Formatted Logical Records,
+defined in Chapter 3 of rp66v1. Each file is an EFLR component, with exception
```

Comment:
Not fully true.
Object  and object attributes actually create one 'object' entity. 'object' is just a header and 'objattr' are attributes defined for the object.
It was easier to separate them and not to repeat same 'object' header every time, given that it doesn't actually give the tests value.
reprcodes go to the same - they also are objattr

plus also with the exception of implicit stuff